lib/Components/Header/header.js

```diff
@@ -0,0 +1,5 @@
+const headerLoginBtn = document.getElementById("sc-login");
+
+headerLoginBtn.addEventListener("click", () => {
+  window.location = "/lib/components/auth/auth.html";
+});
```

Comment:
Why are we using this way of changing URL and not directly changing URL through href in anchor tag?

Reply:
+1

Reply:
yeah it can be done through hrefs also , just working out through buttons